doc/Caliptra.md

```diff
@@ -1006,8 +1010,8 @@ This section describes Caliptra error reporting and handling.
 
 | | Fatal errors | Non-fatal errors |
 | :- | - | - |
-| Hardware | - ICCM, DCCM SRAM ECC<br> - Second watchdog (WD) timer expiry. The first timer expiry triggers an NMI to firmware to correct the issue and clear the interrupt status bit. If the WD expires again, then it is escalated to a FATAL error. | - Mailbox SRAM ECC (except initial firmware load)<br> - Mailbox incorrect protocol or commands. For example, incorrect access ordering or access without Lock. |
-| Firmware | - Boot-time firmware authentication failures<br> - Firmware triggered FATAL errors. Examples: ICCM or DCCM have misaligned access (potentially in the except subroutine); AHB access hangs, triggered through WD timer expiry; AHB access outside of the decoding range; and stack overflow errors. For more information, see the table below. | - Mailbox API failures<br> - First WD timer expiry<br> - Cryptography processing errors |
+| Hardware | - ICCM, DCCM SRAM ECC.<p> </br>-  The second watchdog (WD) timer expiry triggers an NMI, and a FATAL error is signaled to the SoC.</p> | <p>- Mailbox SRAM ECC (except initial firmware load)<br><p>- Mailbox incorrect protocol or commands. For example, incorrect access ordering or access without Lock. |
```

Comment:
remove `<p>` and `</p>` tags from all table updates

Reply:
@rusty1968 , if removing the <p> tags, you will need to separate these into multiple entries in the table

Reply:
removed